lib/Froxlor/Cli/Action/ConfigServicesAction.php

```diff
@@ -83,6 +83,12 @@ private function createConfig()
 		$distros = glob($config_dir . '*.xml');
 		// tmp array
 		$distributions_select_data = array();
+
+		//read os-release
+		$os_dist = parse_ini_file('/etc/os-release', false);
+		$os_version = explode('.',$os_dist['VERSION_ID'])[0];
```

Comment:
you should at least validate that `$os_dist` is an array and that `['VERSION_ID']` exists...pretty ugly if (why so ever) `/etc/os-release` does not exist

Reply:
errh.. yeah

Reply:
/etc/os-relase is a LSB file,
this is required when enabling the froxlor repository's.
the file should always be there.
nevertheless i added a check for it.

Reply:
I know...still, froxlor can basically be installed and used on every system that runs a decent webserver, php and mysql/mariadb database....so it's just in case :)